local-cluster/src/Lib.hs

```diff
@@ -81,11 +81,11 @@ withBFTNode clusterTracer cfg action = do
     traceWith clusterTracer $ MsgNodeStarting cfg
     action rn
  where
-  dlgCertFilename id = "delegation-cert.00" <> show (id - 1) <> ".json"
```

Comment:
So we have an `id` again? :D

Reply:
Yes :no_mouth: ... arguably, this isn't an improvement.